package.json

```diff
@@ -24,7 +24,7 @@
   "scripts": {
     "preinstall": "./scripts/download-mapbox-gl-native-ios.sh 3.2.0",
```

Comment:
I guess so. Didn't test yet. Let's merge though.
On Wed, 20 Jul 2016 at 18:19 Bobby Sudekum <notifications@github.com> wrote:

> In package.json
> <https://github.com/mapbox/react-native-mapbox-gl/pull/365#discussion_r71545932>
> :
>
> > @@ -24,7 +24,7 @@
> >    "scripts": {
> >      "preinstall": "./scripts/download-mapbox-gl-native-ios.sh 3.2.0",
>
> Can we change this to v3.3.1?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mapbox/react-native-mapbox-gl/pull/365/files/d288f274da1f528f6d4fce856b999ecadc240bab#r71545932>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAaIHI-NlG895jSWXHFxuR0lyw3WIHYqks5qXjyGgaJpZM4I4ROX>
> .
>